### PR TITLE
Add setup.md

### PR DESCRIPTION
**Pull request by cdrage**
Adds setup.md to the docs folder in order to sync with changes to
http://kompose.io site (whenever we update setup.md here, it'll update
on the gh-pages branch).

### CODE REVIEW

**Comment by kadel** [comment]
This is creating duplication with README.md :smirk: 
How we are going to keep those in sync?

**Comment by cdrage** [comment]
@kadel Yup, it's a duplication of README.md which is edited when we do a release (version change).

The doc syncs with the setup.md in gh-pages as well.

Yes, we need to update README.md to reflect the new doc.

**Comment by kadel** [comment]
I meant how we are going to keep setup.md and readme in sync.

Its going to be hard  to maintain  install instruction in two places :-(
We are already bad in keeping docs up to date and remembering it to edit it in two places will just add to it :-(

**Comment by kadel** [comment]
It is better to have install instruction in readme, but rather that having it in two places we could just link to setup.md from readme

**Comment by cdrage** [comment]
@kadel Yup, that's what I meant. In the README.md we'll have our recommend way (installation via binary), in the `docs` section, we'll have the elaborate way (CentOS, Fedora, etc.) as we get more packaging and other installation methods added.

**Comment by cdrage** [comment]
@kadel Ready for review, I ended up re-working the installation methods on our README.md to emphasize binary installation as well as have the setup.md document list *all* of our installation methods (much cleaner than our current README)

File: README.md [hunk 1]
```diff
@@ -35,7 +35,16 @@ Other examples are provided in the _examples_ [directory](./examples).
 
 We have multiple ways to install Kompose. Our prefered method is downloading the binary from the latest GitHub release.
 
-#### GitHub release
+Our entire list of installation methods are located in our [setup.md](/docs/setup.md) document.
+
+Installation methods:
+  - [Binary (Prefered method)](README.md)
+  - [Go](/docs/setup.md)
```
Comment (kadel) [inline README.md]: /docs/setup.md#go

File: README.md [hunk 2]
```diff
@@ -35,7 +35,16 @@ Other examples are provided in the _examples_ [directory](./examples).
...
+  - [CentOS](/docs/setup.md)
```
Comment (kadel) [inline README.md]: /docs/setup.md#centos

File: README.md [hunk 3]
```diff
@@ -35,7 +35,16 @@ Other examples are provided in the _examples_ [directory](./examples).
...
+  - [Fedora](/docs/setup.md)
```
Comment (kadel) [inline README.md]: /docs/setup.md#fedora

File: README.md [hunk 4]
```diff
@@ -35,7 +35,16 @@ Other examples are provided in the _examples_ [directory](./examples).
...
+  - [macOS (Homebrew)](/docs/setup.md)
```
Comment (kadel) [inline README.md]: /docs/setup.md#mac-os-x

**Comment by cdrage** [comment]
@kadel updated!

**Pull request merged**